### PR TITLE
ffka-gluon-web-usb-wan-hotplug: add package

### PR DESCRIPTION
**Pull request by herrbett**
This package allows users to enable and disable the FFDA USB-WAN-Hotplug package via Gluon web / the config mode.

### CODE REVIEW

File: ffka-gluon-web-usb-wan-hotplug/i18n/en.po [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+msgid "Enabled"
+msgstr "Enabled"
+
+msgid "USB-WAN-Hotplug"
+msgstr "USB-WAN-Hotplug"
+
+msgid "ffka-gluon-web-usb-wan-hotplug:description"
+msgstr ""
+"Your node can automatically use a USB connected Ethernet adapter as a WAN.
```
Comment (blocktrron) [inline ffka-gluon-web-usb-wan-hotplug/i18n/en.po:21]: Missing terminating `"`

**Comment by blocktrron** [comment]
For the lua linting errors, please rebase on  latest master. I've imported luacheck rules from Gluon to work with gluon web modules.

**Comment by herrbett** [comment]
added terminating ", all checks are now running through smoothly

**Comment by blocktrron** [comment]
@rubo77 Use USB provided WAN (e.g. iPhone / Anroid) to provide an uplink on the go.

**Comment by rubo77** [comment]
And what is the difference to just using the WAN port you already have on most models?

**Comment by herrbett** [comment]
Does your generic iPhone/Android Phone come with a build-in ethernet port? I figure, most models don't, so this package is a neat way of utilising the privat hotspot function of your smartphone or other USB Ethernet Device as an uplink for your Gluon enabled Freifunk device.

**Comment by rubo77** [comment]
Ah OK, now it makes sense.

I clarified this in the explanation: https://github.com/freifunk-gluon/community-packages/pull/8

**Pull request merged**